content/rs/release-notes/rs-7-2-4-releases/rs-7-2-4-52.md

```diff
@@ -520,11 +520,13 @@ REST API requests deprecated as of Redis Enterprise v7.2:
 
 #### Access control deprecations
 
-- The following predefined roles and Redis ACLs are not available after upgrading to Redis Enterprise Software version 7.2.4 if they are not associated with any users or databases in the cluster:
+- The following predefined roles and Redis ACLs are no longer available with new Redis Enterprise Software version 7.2.4 clusters. if they are not associated with any users or databases in the cluster:
```

Comment:
We can probably remove the second part if we're only talking about new clusters.
```suggestion
- The following predefined roles and Redis ACLs are no longer available for new Redis Enterprise Software version 7.2.4 clusters:
```